app/models/sample.rb

```diff
@@ -747,7 +748,13 @@ def kickoff_pipeline
 
     pr.alignment_config = AlignmentConfig.find_by(name: alignment_config_name) if alignment_config_name
     pr.alignment_config ||= AlignmentConfig.find_by(name: AlignmentConfig::DEFAULT_NAME)
-    pr.save
+    pr.save!
+  rescue StandardError => err
```

Comment:
It could be for `ActiveRecord::RecordInvalid` only but I was thinking we would want to handle any error the same way. I don't see cases where we would want to fail silently.

Reply:
You want fail silently anymore since you are using `!`. It is just how much noise you make... Typically, you use the more narrower to provide a better error message. In this case, I agree that it might not be very important as long as err.inspect provides all info.

Reply:
I agree. By fail silently, I was referring to the lack of user facing message.